denizen_scripts/survival/repo-link/MobScripts/SpiderStuff.dsc

```diff
@@ -0,0 +1,10 @@
+Paranoia_poison_canceller:
+  type: world
+  debug: false
+  events:
+    on player damaged by cave_spider:
+      - if <context.damager.is_mythicmob> && <context.damager.entity> == cave_spider:
+#Using determine cancelled then hurt to prevent poison on the player from hallucination spiders.
+        - determine passively cancelled
+        - if <context.damage.entity.mythicmob.stance> == HALLUCINATED
+        - hurt <context.damager.mythicmob.level.sub[6]> <player>
```

Comment:
Missing colon on `if` statement
There's nothing within the `if` statement, but based on the commentary I assume you want to hurt the player directly, so you should indent the command used here. Ideally, however, you should prefer to `determine` the damage, not indirectly `hurt` them instead
```suggestion
        - if <context.damage.entity.mythicmob.stance> == HALLUCINATED:
          - determine <context.damager.mythicmob.level.sub[6]>
```